src/App.js

```diff
@@ -0,0 +1,21 @@
+import './App.css';
+import {
+  BrowserRouter, Route, Link, Routes,
+} from 'react-router-dom';
+import Categories from './components/catagories';
+import Books from './components/books';
+
+function App() {
+  return (
+    <BrowserRouter>
+      <Link to="/">Books</Link>
+      <Link to="/Categories">Categories</Link>
```

Comment:
- [ ] Please create a <`Navbar>` component, which has 2 `<Link>`s. One to `/` and one to `/categories` as per requirement 👍